plugins/ahmadfatoni/apigenerator/controllers/api/UsersAPIController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AhmadFatoni\ApiGenerator\Controllers\API;
+
+use Cms\Classes\Controller;
+
+use Illuminate\Http\Request;
+use AhmadFatoni\ApiGenerator\Helpers\Helpers;
+use Tymon\JWTAuth\Facades\JWTAuth;
+use RainLab\User\Models\User;
+
+class UsersAPIController extends Controller
+{
+    protected $User;
+
+    protected $helpers;
+
+    public function __construct(User $User, Helpers $helpers)
+    {
+        parent::__construct();
+        $this->User    = $User;
+        $this->helpers          = $helpers;
+    }
+
+
+    public function index()
+    {
+        $data = $this->User->select('id', 'name', 'email')->get()->toArray();
```

Comment:
Nếu user là collection rồi thì ông test như này xem kết quả có như ông đang cần không nhé
```suggestion
        $data = $this->user->pluck('id', 'name', 'email');
```
hoặc nếu không được thì thử như này 
```suggestion
        $data = $this->user->pluck('id', 'name', 'email')->toArray();
```

Reply:
À, hiện cái route index này t khóa lại rồi. Vì mình ko cho phép ng dùng gọi API lấy danh sách user đâu :v

Reply:
Nmà thanks ô nhé, t sẽ thử cái đó